### PR TITLE
PR for final output on setting up linters output

### PR DESCRIPTION
**Pull request by iAmao**
### Peer Reviewers

Name :  Micheal Ogala
Comment : Excellent setup. Good use of screenshots to describe PR.

Name : Moyosore Sosan
Comment : Set your eslint configuration to flag out the use of 'var'. You should use es6 'let' or 'const' keyword for variable declaration. Also make use of es6 arrow function for function declaration

---

#### What does this PR do?
This PR contains completed output for setting up eslinter and Houndci for project.
#### Description of Task to be completed?
Task enables checking of Airbnb ES6 style violation on command line and HoundCI comment on a PR
#### How should this be manually tested?
```
git clone https://github.com/andela-iamao/inverted-index.git
cd inverted-index
npm install
./node_modules/.bin/eslint dist/js/eslint.test.js
```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Set up eslint

#### Screenshots 

Alt text
[Screenshot 1](https://raw.githubusercontent.com/q-ode/test-simulations-mgmt-repo/bayo-izanami-iamao/06-building-a-program-start-to-finish/setting-up-linters/output/screenshot1.png)

Alt text
[Screenshot 2](https://raw.githubusercontent.com/q-ode/test-simulations-mgmt-repo/bayo-izanami-iamao/06-building-a-program-start-to-finish/setting-up-linters/output/screenshot2.png)

#### Questions:
N/A

### CODE REVIEW

**Comment by q-ode** [comment]
Kindly resolve conflicts.

**Comment by iAmao** [comment]
Fix added for merge conflicts

**Review by q-ode — COMMENTED** [review]
The hosting service you used for the images is blocked on this network.

Kindly host on a platform I can access.

**Comment by iAmao** [comment]
That's done now its hosted on google photos

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Link to github repo
+- [andela-iamao/inverted-index](https://github.com/andela-iamao/inverted-index)
+
+#### Screenshots
+
+- ##### Eslint [Airbnb](https://github.com/airbnb/javascript) violation on terminal
+ - ![eslint terminal screenshot](https://lh3.googleusercontent.com/vvyTNCfX24bf06akkqkKJ-DR9yzXUCSp9qZ9K5jui_dq-oK08-hEAxEUj-nm6DXJXL93oDgIm_pe1Vqe1crOkYLUJWu4qV4Il1Waj5xGUVL_cfFWpZDvA01ls5EAM8x7jTUrUP6oHF6qu-4nY1wkQR3af7hV08MxZSB-ckTqha6bQJwc40-Zu-y51Irj_2A0vN_xv9VVhz5SQ0RkbveJ6JppGCok-53JSMc2859xERvOAn8Us_aNo_hDyMs30_jPgfVgaX3ZgIYp7yoTcvwSYx23nIOGvnjanG9JjNFNNJON873eM381lqTOWu1R4qdlTZfazHjCLtyXrd6tWseG3L9HBb1sE5m_E3IvLLY8XRio04HNzEcVqHYeH3NVEzr7DFRN5ukMHsh5aN7tuOsdfWH7qAN-navP9qZEHY4FiXQTCQ_haPAt7nO-ZJDlJzjXsq-mhgWEUGacW0qOPoHy9jIValbg4ZyNP_vKVLDtqP1P2cepcnykIihjBYw9XX_k2ZcsjWDUExCBZwHleRCJCB-p2D7giHHfcZfQuyjgZyn8KFiklPSC1kT-gXrEMhc3GsX1lMzIc5hpYjDg95m4kOdH1ld1yXAKHClb2U6PIAOWU40=w1178-h662-no "Screenshot 1")
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: What are you trying to achieve here?

I see no images.

Comment (iAmao) [reply 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: I have pushed the images directly to the output folder

**Pull request merged**